.travis.yml

```diff
@@ -45,6 +45,10 @@ matrix:
     fast_finish: true
     include:
 
+        # Try with pint
+        - python: 3.6
```

Comment:
are we only supporting 3.6 and 2.7 at the moment, or am I misunderstanding this?

And should pint not work under 2.7?